tools/ogs_tools/.shed.yml

```diff
@@ -0,0 +1,20 @@
+name: suite_ogs_tools
```

Comment:
We need to adapt the names, descriptions, long_descriptions a bit. We can discuss next week or by zoom.